### DISCUSSION TITLE
电汽朋克 - Miao Yu | 于淼

### DISCUSSION THREAD

**Discussion by giscus** [yufree/yufree.cn#459] [General]
# 电汽朋克 - Miao Yu | 于淼

https://yufree.cn/cn/2024/02/23/autopunk/

**Comment by hongtaoh** [comment]
难以想象你居然在美国不开车。但后来一想，其实不开车也没什么，而且你在纽约，更没有必要。

你说的未来的交通工具，我第一想到了三体第二部里的伞。

**Reply by yufree** [reply]
额，我已经搬出纽约两年多了，现在住康州荒郊野外，门口两个公交车站倒也够用。
